worlds/oot/Patches.py

```diff
@@ -208,8 +208,8 @@ def patch_rom(world, rom):
 
     # Fix Ice Cavern Alcove Camera
     if not world.dungeon_mq['Ice Cavern']:
-        rom.write_byte(0x2BECA25,0x01);
```

Comment:
HELP!?!?!!!!?!??!??!?!?!?! I NEED ESPEON TO EXPLAIN HIMSELF RIGHT NOW

Reply:
About to label this `release/blocker` over this oh my lord

Reply:
This is your yearly reminder that `;` is perfectly legal in Python and is just ignored. Apparently it's meant for multiline statements but is fine with the other statement being null?